.changeset/itchy-avocados-peel.md

```diff
@@ -0,0 +1,7 @@
+---
+'skuba': minor
+---
+
+lint: Manage `.npmrc` for pnpm projects
+
+skuba now manages `.npmrc` when a project uses `pnpm` to enable [dependency hoisting](https://pnpm.io/npmrc#dependency-hoisting-settings). It will attempt to strip sensitive auth tokens from the file, and remove `.gitignore` entries that ignore `.npmrc`.
```

Comment:
In lieu of a pnpm migration guide, maybe we can leave this like so for the time being:

```suggestion
skuba now manages a section of `.npmrc` when a project uses `pnpm` to enable [dependency hoisting](https://pnpm.io/npmrc#dependency-hoisting-settings). It will continue to avoid committing autofixes to the file if it contains auth secrets.
```